packages/trader/src/sass/app/modules/mt5/cfd-dashboard.scss

```diff
@@ -1153,8 +1114,62 @@
         padding-top: 0.8rem;
         position: relative;
 
-        li {
-            margin: 0 0 0.8rem 0;
+        .account-poa__upload-section {
```

Comment:
Suggestion: This may be easier to trace back but better to follow the code standard in Deriv app and use nested pseudo-classes since we are using `SCSS` than basic `CSS`